homeassistant/components/squeezebox/media_player.py

```diff
@@ -65,79 +67,118 @@
     | SUPPORT_CLEAR_PLAYLIST
 )
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We should leave the platform schema and add `cv.deprecated` so they still get validation of the yaml during the transition period.

Reply:
Ah, got it. And now I see where cv.deprecated goes. Thanks.